### PR TITLE
Rename AddArtist, Artist UI folders. Other UI Work

### PR DESCRIPTION
**Pull request by Qstick**
#### Database Migration
NO

#### Description
Initial work to separate AddArtist View and Artist Views (used to be Add Series/Series). Fixes for Nav Search, Fixes for Artist Editor View. Many other misc wording/functional tweaks. Hopefully many things are less broken, and nothing new is broken :)

#### Todos
- [ ] Tests

#### Issues Fixed or Closed by this PR

*

### CODE REVIEW

File: src/UI/AddArtist/EmptyViewTemplate.hbs [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<div class="text-center hint col-md-12">
+    <span>You can also search by iTunes using the itunes: prefixes.</span>
```
Comment (majora2007) [inline src/UI/AddArtist/EmptyViewTemplate.hbs]: This should by Spotify by using spotify: or spotifyId: prefixes.

File: src/UI/AddArtist/Existing/AddExistingArtistCollectionViewTemplate.hbs [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<div class="x-existing-folders">
+    <div class="loading-folders x-loading-folders">
+        Loading search results from iTunes for your artists, this may take a few minutes.
```
Comment (majora2007) [inline src/UI/AddArtist/Existing/AddExistingArtistCollectionViewTemplate.hbs]: We are using Spotify, please update this.

File: src/UI/Artist/ArtistController.js [hunk 3]
```diff
@@ -17,18 +17,18 @@ module.exports = NzbDroneController.extend({
 
     artist : function() {
         this.setTitle('Lidarr');
-        this.showMainRegion(new SeriesIndexLayout());
+        this.showMainRegion(new ArtistIndexLayout());
     },
 
     artistDetails : function(query) {
-        var artists = ArtistCollection.where({ artistNameSlug : query });
+        var artists = ArtistCollection.where({ artistSlug : query });
```
Comment (majora2007) [inline src/UI/Artist/ArtistController.js]: I believe this needs to be artistNameSlug. Can you confirm?

File: src/UI/AddArtist/EmptyViewTemplate.hbs (hunk 1 above)
Comment (Qstick) [inline src/UI/AddArtist/EmptyViewTemplate.hbs]: Yep, I will update

**Comment by Qstick** [comment]
Updated iTunes to Spotify for now to get this thru. Will change again once we change to new server.

**Pull request merged**